utilities/unittests/test_console.py

```
@@ -400,11 +408,240 @@ def test_console_eof_sampler_no_sample(self, mock_get_dir, mock_timeout_sampler)
+    @patch("console.os.close")
+    @patch("console.pexpect.fdpexpect.fdspawn")
+    @patch("console.subprocess.Popen")
+    @patch("console.pty.openpty")
+    @patch("console.get_data_collector_base_directory")
+    def test_spawn_console_fdspawn_failure_force_kill(
+        self,
+        mock_get_dir,
+        mock_openpty,
+        mock_popen,
+        mock_fdspawn,
+        mock_os_close,
```

Ruff (0.15.15)

[warning] Unused method argument: `mock_os_close`

(ARG002)